core/src/main/java/software/amazon/awssdk/core/util/UserAgentUtils.java

```diff
@@ -29,8 +29,8 @@
 @ThreadSafe
 public final class UserAgentUtils {
 
-    private static final String UA_STRING = "aws-sdk-{platform}/{version} {os.name}/{os.version} {java.vm.name}/{java.vm.version}"
-                                            + "/{java.version}{language.and.region}{additional.languages}";
+    private static final String UA_STRING = "aws-sdk-{platform}/{version} {os.name}/{os.version} {java.vm.name}/{java.vm"
+                                            + ".version} Java/{java.version}{language.and.region}{additional.languages}";
```

Comment:
I like it, but we should check to make sure this won't cause any metrics issues.

Reply:
good point, will check with them.

Reply:
Changes look good to me.